### PR TITLE
use OWNER aliases vs. specific people

### PR DESCRIPTION
**Pull request by dprotaso**

### CODE REVIEW

File: label_sync/OWNERS [hunk 1]
```diff
@@ -1,10 +1,8 @@
 # The OWNERS file is used by prow to automatically merge approved PRs.
 
 approvers:
-- steering-committee
```
Comment (dprotaso) [inline label_sync/OWNERS:4]: already inherited from top level OWNER files

File: groups/OWNERS [hunk 2]
```diff
@@ -8,9 +8,9 @@ options:
   no_parent_owners: true
 
 approvers:
-- dprotaso
-- upodroid
-- kvmware
```
Comment (dprotaso) [inline groups/OWNERS:13]: adding TOC here so it's not just me :)

**Review by upodroid — APPROVED** [review]
LGTM

**Pull request merged**